### PR TITLE
branson/refactor

### PR DESCRIPTION
**Pull request by BransonNg**

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -11,6 +11,18 @@
     public static final String MESSAGE_INVALID_TASK_TO_BE_DONED =
             "Task has already been marked as done!";
     public static final String MESSAGE_FIELD_UNKNOWN = "The fields do not exist %1$s";
+    // Command completor messages
+    public static final String COMPLETE_SUCCESS = "Message auto completed: ";
```
Comment (artlee06) [inline src/main/java/seedu/address/commons/core/Messages.java:15]: Good effort in adding some feedback messages for the user!

File: src/main/java/seedu/address/logic/commands/CommandCompletor.java [hunk 2]
```diff
@@ -49,138 +39,52 @@ public CommandCompletor() {
...
-        boolean hasPriority = ParserUtil.arePrefixesPresent(argMultimap, PREFIX_PRIORITY);
-        String prefixesAdded = "", feedbackToUser = UNCHANGED_SUCCESS;
+    public CompletorResult getSuggestedCommand(String input, int listSize)
+            throws CompletorException {
+        String[] splitInput = input.split("\\s+");
+        String feedbackToUser = Messages.COMPLETE_SUCCESS;
 
-        if (trimmedInputWords.length <= 0) {
-            throw new CompletorException(COMPLETE_FAILURE_COMMAND);
+        if (splitInput.length <= 0 || splitInput[0].length() == 0) {
```
Comment (artlee06) [inline src/main/java/seedu/address/logic/commands/CommandCompletor.java:56]: Hmmmm what is wrong with <=  ?

File: src/main/java/seedu/address/ui/CssManipulator.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package seedu.address.ui;
+
+import javafx.collections.ObservableList;
+import javafx.scene.Parent;
+
+public class CssManipulator {
```
Comment (artlee06) [inline src/main/java/seedu/address/ui/CssManipulator.java:6]: Interesting how you added a CSS manipulator to display the different states and thus styling

File: src/main/java/seedu/address/ui/ResultDisplay.java [hunk 4]
```diff
@@ -10,15 +13,32 @@
...
+    public void setWarning() {
+        CssManipulator.setStyleToIndicateWarning(resultDisplay);
+        scheduler.cancel();
+        scheduler.purge();
+        scheduler = new Timer();
+        scheduler.schedule(
+                new TimerTask() {
+                    @Override
+                    public void run() {
+                        Platform.runLater(() -> CssManipulator.setStyleToDefault(resultDisplay));
```
Comment (artlee06) [inline src/main/java/seedu/address/ui/ResultDisplay.java:39]: Interesting so after 1 second time delay it changes back to original styling? But shouldn't it remain as a warning style until something

**Review by artlee06 — APPROVED** [review]
Is a huge refactor, have some minor questions stated in the review but just slightly concerned with whether it affects the rest after merging it in

**Comment by BransonNg** [comment]
> Hmmmm what is wrong with <=  ? 

Needa address the case where splitInput is empty or first element of splitInput is an empty string

**Comment by BransonNg** [comment]
> Interesting so after 1 second time delay it changes back to original styling? But shouldn't it remain as a warning style until something  

hahah that means i need to get the CommandBox to manipulate ResultDisplay's CSS. It might be better, am also considering it

**Pull request merged**